cedar-policy/src/api.rs

```diff
@@ -2373,6 +2384,38 @@ impl PolicySet {
     /// rules.  Policy formatting can be done through the Cedar policy CLI or
     /// the `cedar-policy-formatter` crate.
     pub fn to_cedar(&self) -> Option<String> {
+        match self.to_string_representations() {
+            Some(StringifiedPolicySet {
+                policies,
+                policy_templates,
+            }) => {
+                let policies_as_vec = policies
+                    .into_iter()
+                    .chain(policy_templates)
+                    .collect::<Vec<_>>();
+                Some(policies_as_vec.join("\n\n"))
+            }
+            None => None,
+        }
+    }
+
+    /// Get the human-readable Cedar syntax representation of this policy set,
+    /// as a vec of strings. This function is useful to break up a large cedar
+    /// file containing many policies into individual policies.
+    ///
+    /// This will return `None` if there are any linked policies in the policy
+    /// set because they cannot be directly rendered in Cedar syntax. It also
+    /// cannot record policy ids because these cannot be specified in the Cedar
+    /// syntax. The policies may be reordered, so parsing the resulting string
+    /// with [`PolicySet::from_str`] is likely to yield different policy id
+    /// assignments. For these reasons you should prefer serializing as JSON (or protobuf) and
+    /// only using this function to obtain a representation to display to human
+    /// users.
+    ///
+    /// This function does not format the policy according to any particular
+    /// rules.  Policy formatting can be done through the Cedar policy CLI or
+    /// the `cedar-policy-formatter` crate.
+    pub fn to_string_representations(&self) -> Option<StringifiedPolicySet> {
```

Comment:
"representations" may be too general here, have you considered other names like `to_stringified_policyset` or `stringify`? Normally, I'd suggest `try_...` since the conversion can fail, but we didn't follow this convention with `to_cedar`.